app/views/application/_navbar.html.erb

```diff
@@ -1,19 +1,20 @@
 <%# Bootstrap navbar helper provided by twitter-bootstrap-rails gem  %>
 <%# https://github.com/seyhunak/twitter-bootstrap-rails#navbar-helper %>
-<%= nav_bar brand: t('appname'), responsive: true do %>
+
+<%= nav_bar brand: t('appname'), responsive: true, fluid: true do %>
   <%= menu_group do %>
     <%= menu_item User.model_name.human(:count => :many).titleize, users_path %>
     <%= menu_item Team.model_name.human(:count => :many).titleize, teams_path %>
     <%= menu_item Match.model_name.human(:count => :many).titleize, matches_path %>
     <%= menu_item Event.model_name.human(:count => :many).titleize, events_path %>
-    <%= menu_divider %>
+  <% end %>
+  <%= menu_group pull: :right do %>
     <% if user_signed_in? %>
       <%= menu_item current_user.first_name, user_path(current_user) %>
       <%= menu_divider %>
```

Comment:
The devider messes up the layout on mobile:
![screen shot 2017-11-24 at 16 43 23](https://user-images.githubusercontent.com/15202881/33217067-c6c4153c-d136-11e7-8637-07dce74b710a.png)

Reply:
Shouldn't be a big deal anymore after incorporating changes from `dev` because the 'Abmelden' button has moved into a dropdown menu by now.